src/engagement_db_to_analysis/engagement_db_to_analysis.py

```diff
@@ -164,6 +424,14 @@ def generate_analysis_files(user, pipeline_config, engagement_db, cache_path=Non
 
     messages_traced_data = filter_messages(user, messages_traced_data, pipeline_config)
 
-    participants_traced_data_map = _fold_messages_by_uid(user, messages_traced_data)
+    messages_by_column = _convert_to_messages_column_format(user, messages_traced_data, pipeline_config.analysis_configs)
```

Comment:
Does it make sense to move the other helper functions to a traced_data_conversion library? That way we can reuse some of them in other scripts e.g `_analysis_dataset_config_for_message`....

Reply:
agreed. I think better to do this in a follow-up PR so it's easier to review the move